network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -0,0 +1,24 @@
+{% extends "./modular_page.html" %}
+
+{% load wagtailcore_tags mini_site_tags %}
+
+{% block pre_footer_classes %}{% if singleton_page %}col-12 col-lg-10 offset-lg-1{% else %}col-md-12{% endif %}{% endblock %}
+
+{% block subcontent %}
+<div class="py-4 py-md-5
+  {% if page.narrowed_page_content == True %}
```

Comment:
The page is always narrow, why have this check? I believe/hope the plan is to move away from the narrowed page option in general and be more opinionated.

Reply:
Related to that, if the page content is always narrow, then we probably don't need the model to specify that value at all. The template can simply be written to match the fact that blog content is always narrow (with the same comment for zen mode)